docs/Dockerfile

```diff
@@ -1,28 +0,0 @@
-FROM python:rc-alpine3.12 as builder
```

Comment:
Can we keep the dockerfile to spin up a local version of the docs site? It allows us to validate that the docs are correct before pushing to git

Reply:
You can have a local version just by running
`npm run start` 

Do you still think we need the dockerfile for it?